.readthedocs.yml

```diff
@@ -5,21 +5,21 @@
 # Required
 version: 2
 
+# Set the version of Python and other tools you might need
 build:
-  image: latest # Python 3.8 available on latest
-  apt_packages:
-    - pandoc
+  os: ubuntu-20.04
+  tools:
+    python: "3.9"
```

Comment:
What happened with `pandoc`?

Reply:
Thanks, this was an ommision. Weirdly `nbsphinx` still appeared to work OK, but I've added it back in to be safe.